meshtastic/tests/test_node.py

```
@@ -1247,6 +1311,284 @@ def test_setURL_add_only_raises_when_no_disabled_slot_available(
+    primary.settings.name = "primary"
+    primary.settings.psk = b"\x01"
+    disabled1 = Channel(index=1, role=Channel.Role.DISABLED)
+    disabled2 = Channel(index=2, role=Channel.Role.DISABLED)
+    anode.channels = [primary, disabled1, disabled2]
+    anode.localConfig.lora.hop_limit = 3
+
+    send_calls = {"rollback_failures": 0, "stage_writes": 0}
+
+    def _rollback_send_fails_once(
+        msg: admin_pb2.AdminMessage,
+        wantResponse: bool = False,
```

Ruff (0.15.5)

[error] Boolean-typed positional argument in function definition

(FBT001)